db/schema.rb

```diff
@@ -64,9 +64,6 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "image"
-    t.string "image_content_type"
```

Comment:
I'm not sure why these got removed in your branch, it doesn't seem right.

Reply:
This was just from master. It told me i needed to run migrations so i did so. Can you clone a fresh branch down and see?